documentation/releaseNotes/releaseNotes.v6.1.0.md

```diff
@@ -0,0 +1,9 @@
+Today we are releasing the 6.1.0 build of the `dotnet monitor` tool. This release includes:
+
+- Load a profiler via a collection rule action (#781)
+- Get or Set an environment variable via a collection rule action (#1176)
+- Add `MachineJson` option to `--output` parameter for `generatekey` command. This is useful for automating credential provisioning. (#1428)
+- Allow for simplified diagnostic port configuration (#1292)
+- Ensure `config show` command displays arrays without indices (#1054)
+
+\*⚠️ **_indicates a breaking change_**
```

Comment:
Could you remove the breaking changes line since there are no breaking changes? I typically remove it after the release has gone up because we forget to remove it. I remove it so users aren't trying to figure out what breaking changes there are even though there aren't any.